src/transport_ofi.h

```diff
@@ -120,7 +120,9 @@ extern shmem_free_list_t *shmem_transport_ofi_frag_buffers;
 #define OFI_ABORT(...) { fprintf(stderr, __FILE__ ":%d: \n", __LINE__); \
                             fprintf(stderr, __VA_ARGS__);  }
 
-#define OFI_RET_CHECK(ret) do { if (ret) { RAISE_ERROR(ret); } } while (0)
+#define OFI_RET_CHECK(ret) do { if (ret) { \
+	fprintf(stderr,"OFI error #%d: %s \n", (int)ret, fi_strerror(ret)); \
```

Comment:
Is ret ever not an int?  I would prefer to drop this type cast if we can.  Type casts can mask bugs.